lib/validation.js

```diff
@@ -20,8 +20,12 @@
  */
 
 module.exports = function textareaLimitedValidation(input, settings) {
-  if (input.target.value === '' && !settings.target.empty) {
-    return `${input.target.name} cannot be left blank!`;
+  if (input.target.value && settings.hasOwnProperty('target') && settings.target.hasOwnProperty('minlength') && (settings.target.minlength > input.target.value.length)) {
```

Comment:
Both of these validations should be wrapped under the same shared `if`, so `input.target.value`